src/Umbraco.Web.UI.Client/src/views/logviewer/overview.controller.js

```diff
@@ -10,11 +10,12 @@
         vm.numberOfErrors = 0;
         vm.commonLogMessages = [];
         vm.commonLogMessagesCount = 10;
+        vm.dateRangeLabel = "";
 
         // ChartJS Options - for count/overview of log distribution
-        vm.logTypeLabels = ["Info", "Debug", "Warning", "Error", "Critical"];
+        vm.logTypeLabels = ["Debug", "Info", "Warning", "Error", "Fatal"];
```

Comment:
If you really want all log levels to be displayed there is a `Verbose` level too in Serilog however I am not sure how useful or not that is (as doubt many really set the log level to that)

So it's
* Verbose
* Debug
* Information
* Warning
* Error
* Fatal

Reply:
I've left verbose out, I agree it's not super useful.